.github/scripts/auto-label.cjs

```diff
@@ -0,0 +1,165 @@
+'use strict';
+const path = require('path');
+
+const babelParser = require('recast/parsers/babel');
+const json5 = require('json5');
+const lodash = require('lodash');
+const recast = require('recast');
+const github = require('@actions/github');
+const { HttpClient } = require('@actions/http-client');
+
+const prefixLabelMap = {
+  'resources/': 'resources',
+  'util/': 'util',
+  'test/': 'test',
+  '.mocharc.cjs': 'test',
+  'eslint/': 'style',
+  '.eslintrc.js': 'style',
+  '.github/workflows/': 'ci',
+  '.github/scripts/': 'ci',
+
```

Comment:
Hmm, one thing I was thinking is that we already have a `fisher` label which is `fishing 🎣`.  I think also `oopsyraidsy` might be better off just being `oopsy` for brevity.  What do you think about listing everything out here explicitly, e.g.

```suggestion
  'ui/config/': 'config',
  'ui/dps/': 'dps',
  'ui/eureka/': 'eureka',
  'ui/fisher/': 'fishing🎣',
  'ui/jobs/': 'jobs',
  'ui/oopsyraidsy/': 'oopsy',
  'ui/pullcounter/': 'pull',
  'ui/radar/': 'radar',
  'ui/raidboss/': 'raidboss',
  'ui/test/': 'test',
```